src/libraries/icubmod/embObjAnalog/embObjAnalogSensor.ini

```diff
@@ -1,5 +0,0 @@
-[plugin embObjAnalogSensor]
-type device
-name embObjAnalogSensor
-library embObjAnalogSensor
-wrapper analogServer
```

Comment:
Please double check if it is correct `analogServer` or `AnalogServer`

Reply:
fixed